package com.kodilla.exception.test;

public class SecondChallengeException extends Exception{

    public void SecondChallengeExceptionMet() {
        System.out.println("\n" + "SecondChallengeException Method");
        System.out.println("Ready for action");
    }
}
